pkg/controller/shoot/shoot_control_reconcile.go

```diff
@@ -313,36 +319,45 @@ func (c *defaultControl) updateShootStatusReconcileError(o *operation.Operation,
 		description   = lastError.Description
 		lastOperation = o.Shoot.Info.Status.LastOperation
 		progress      = 1
+		willRetry     = utils.TimeElapsed(o.Shoot.Info.Status.RetryCycleStartTime, c.config.Controllers.Shoot.RetryDuration.Duration)
```

Comment:
Shouldn't this be 
```suggestion
		willNotRetry     = utils.TimeElapsed(o.Shoot.Info.Status.RetryCycleStartTime, c.config.Controllers.Shoot.RetryDuration.Duration)
```

or

```suggestion
		willRetry     = !utils.TimeElapsed(o.Shoot.Info.Status.RetryCycleStartTime, c.config.Controllers.Shoot.RetryDuration.Duration)
```